src/features/Apiexplorer/Dropdown/Dropdown.module.scss

```diff
@@ -93,17 +93,3 @@
     }
   }
 }
```

Comment:
please don't remove these for now, since we may get dark theme in the future sprints. thanks :pray:

Reply:
Reverted